tools/scylla-nodetool.cc

```diff
@@ -3742,6 +3978,140 @@ For more information, see: {}"
                 table_stats_operation
             }
         },
+        {
+            {
+                "tasks",
+                "Manages task manager tasks",
+                "",
+                { },
+                {
+                    typed_option<sstring>("command", "The uuid of a task", 1),
+                },
+                {
+                        {
+                            "abort",
+                            "Aborts the task",
+                            fmt::format(R"(
+                            For more information, see: {}"
+                            )", doc_link("operating-scylla/nodetool-commands/tasks/abort.html")),
```

Comment:
Please add some more details here. E.g. mention what happens if you call this with task which cannot be aborted.

Note that although all commands have link to online docs, we should also have something in this description too.
The description of the command in `scylla-nodetool`, should contain information, that is needed to run the command. E.g., no need to explain here what tasks are, but you need to mention what happens if you pass uuid of non-abortable task here.
The online doc will go into detail on what tasks are.

Think of this description, as a quick reference for expert users, and the online doc as an exhaustive document for people new to tasks API.

Reply:
Also applies to the other descriptions below.